spring-boot/admin/src/main/resources/templates/admin/fragments/services/edit.html

```diff
@@ -51,6 +51,14 @@ <h2 th:text="#{admin.services.edit.form.title}">
                                 <div class="form-group">
                                     <label class="control-label col-md-3 col-sm-3 col-xs-12">
                                         <span th:text="#{admin.services.edit.form.label.id}"/>
+                                        <a data-toggle="modal" data-target=".getting-started-service-identifier-modal">
+                                            <i class="fa fa-question-circle"></i>
+                                        </a>
+                                        <a data-toggle="modal" data-target=".getting-started-service-identifier-warning-modal">
+                                            <i class="fa fa-warning" style="color: #e67300"></i>
+                                        </a>
+                                        <br/>
+                                        <small style="color: #e67300">Be careful about editing!</small>
```

Comment:
It is hard to recognize as a user that there are two small icons that can be clicked, next to each other, therefore I would suggest to move the "warning" icon to the next line and also include the text in the link, resulting in the following:
```
<br/>
<a data-toggle="modal" data-target=".getting-started-service-identifier-warning-modal" style="color: #e67300">
        <i class="fa fa-warning"></i>
        <small>Be careful about editing!</small>
</a> 
```

Reply:
Done as suggested.